packages/calendar/src/types.ts

```diff
@@ -209,6 +209,7 @@ export type TimeRangeSvgProps = Dimensions & { data: CalendarDatum[] } & Partial
                 square: boolean
                 role: string
                 weekdayLegendOffset: number
+                weekdayTicks: number[]
```

Comment:
```suggestion
                weekdayTicks: Array<0 | 1 | 2 | 3 | 4 | 5 | 6>
```

Let's update the type to be more strict.

Reply:
Made changes as suggested